tests/MockClientStrategy.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\Tests;
+
+use Http\Client\HttpAsyncClient;
+use Http\Client\HttpClient;
+use Http\Discovery\Strategy\DiscoveryStrategy;
+use Http\Mock\Client as Mock;
+
+/**
+ * This class is ported from Http\Discovery\Strategy\MockClientStrategy.
+ *
+ * This versions provides the Mock client when an asycn client is required too.
```

Comment:
`This version` (without the final `s`). Also it would make sense to open an issue or PR in HTTPlug repository and link it here so that we know we can remove this class when upstream will fix the problem